### PR TITLE
added deleting comments

### PR DESCRIPTION
**Pull request by Gary-Huynh**

### CODE REVIEW

File: api.js
Comment (RichardNeat) [inline api.js]: please remove console logs

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice one Gary - just some considerations please:
- Think about how we can handle this deletion process optimistically so that each user experience has a good experience. Similar to the voting system.
- Consider how we will handle slow connections, currently I can click the delete button multiple times while it's loading and I don't get any messaging to say my request is loading. This returns an error if I click multiple times.
- Consider how the error handling will work to undo the optimistic rendering

**Comment by Beth-McCormack** [comment]
This still isn't very dry, you dont need to have 4 different api calls when sorting. Have a look at the NC notes on axios and how you can use **optional queries** here🙂

**Review by RichardNeat — APPROVED** [review]
Nice one Gary! Looking good now :) Just a couple of things I noticed regarding comment posting that you may want to address:
- Once I've posted one comment, I can't post another one unless I refresh. Is that intentional? 
- I can't press enter to submit my comment
You're good to merge, if you want to consider the above please do so on a separate branch but no need to submit for approval. Just a consideration for you :)

**Pull request merged**
